src/javascripts/menu items/menu.js

```diff
@@ -16,4 +16,24 @@ const showMenuItems = (array) => {
   });
 };
 
-export default showMenuItems;
+const showUserMenuItems = (array) => {
+  document.querySelector('#form-container').innerHTML = '';
+  document.querySelector('#view').innerHTML = '';
+  document.querySelector('#stage').innerHTML = '';
+  document.querySelector('#modal-container').innerHTML = '';
+
+  array.forEach((item) => {
+    document.querySelector('#view').innerHTML += `<div class="media">
+    <img src="${item.image}" class="mr-3" id="menuItemImage"alt="...">
+    <div class="media-body">
+      <h5 class="mt-0">${item.title}</h5>
+      <p>${item.description}</p>
+      <p>${item.price}</p>
+      <button type="button" class="btn btn-secondary" id="edit-menu-item--${item.firebasekey}">Edit</button>
```

Comment:
Remove this edit button. We don't want nonfunctional elements in the production. This is also considered working ahead.